src/content/advanced/gitops/apps/add_app_template.md

```diff
@@ -0,0 +1,103 @@
+---
+linkTitle: Add a new App Template
+title: Add a new App Template
+description: Learn how to create application templates to deploy apps using GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-add-app-template
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I create an template for app deployment in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+This document is part of the documentation to use GitOps with Giant Swarm App Platform. You can find more information about the [App Platform in our docs](/platform-overview/app-platform/).
+
+# Add a new App Template to the repository
```

Comment:
On the  capitalisation of "App Template" and others:

Throughout this file, the term "App Template" is capitalised - this makes me think that there is a type called `AppTemplate` especially when read alongside `Config Map` or `App Custom Resource`.

As `App Template` is not a type, but a pattern used to define a type, I would consider not capitalising this to avoid confusion to the reader.

Reply:
changed everywhere but the title (title case)